tests/Integration/Behaviour/Features/Scenario/CmsPage/cms_page_management.feature

```diff
@@ -0,0 +1,70 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s cms_page
+@reset-database-before-feature
+Feature: CmsPage Management
+  PrestaShop allows BO users to manage cms pages
+  As a BO user
+  I must be able to create, edit and delete cms page in my shop
+
+  Scenario: Adding new cms page
+    When I add new cms page "cmspage-1" with following properties:
+      | meta_title           | test.This is the title field          |
+      | head_seo_title       | headseotitle(legacy) or metatitle(new)|
+      | meta_description     |                                       |
+      | meta_keywords        | test,keyword,key2,key3                |
+      | link_rewrite         | friendlyurlexample                    |
+      | content              | <div> hello world </div>              |
+      | indexation           | 1                                     |
+      | active               | 0                                     |
+    Then cms page "cmspage-1" "meta_title" in default language should be "test.This is the title field"
+    And cms page "cmspage-1" "head_seo_title" in default language should be "headseotitle(legacy) or metatitle(new)"
+    And cms page "cmspage-1" "meta_description" field in default language should be empty
+    And cms page "cmspage-1" "link_rewrite" in default language should be "friendlyurlexample"
+    And cms page "cmspage-1" "content" in default language should be "<div> hello world </div>"
+    And cms page "cmspage-1" indexation for search engines should be enabled
+    And cms page "cmspage-1" should be not displayed
+
+  Scenario: Attempting to create cms page with empty title
+    Given cms page "cmspage-2" does not exist
+    When I attempt to create cms page "cmspage-2" with empty title
+    Then I should get error message 'Cms page contains invalid field values'
```

Comment:
Hmmm, I don't think you should assert for exception message. For me, it seems that this wording would be more correct: ` Then I should get an error that CMS page data is invalid` or something similar, wdyt?

Reply:
But isn't it better to expect exact error? Plus I can reuse the exceptionAssert with other cases. Why it's no good?

Reply:
In unit or integration tests I think it would be ok, but in Behat scenarios I'd prefer to have non technical steps. Although in this case exception message is understandable for non technical people, I still think we should describe step with more user friendly error.

Also, changing exception message should not break build imo, because code hasn't changed, but in this case it would.

> Plus I can reuse the exceptionAssert with other cases.

You can reuse it either way.

@matks I think this comes to your decision.

Reply:
Behat is a integration tests tool :thinking: 
I'm agree with @zuk3975 We must test error message, otherwise they are useless.
Testing an error message is less technical than `with cms category id "5000"` because the user think with name, not id.
Same with `And CMS page "cmspage-1" "head_seo_title" in default language should be 'delivery options'`, if you want to not be technical use:
`And CMS page "seo title" in default language should be 'delivery options'`

Reply:
> Same with `And CMS page "cmspage-1" "head_seo_title" in default language should be 'delivery options'`, if you want to not be technical use:
> `And CMS page "seo title" in default language should be 'delivery options'`

In this case I am using the exact properties name purely to ease reusability and save time, ofc it becomes really technical that way. Must it be refactored to your suggestion, or it's ok?

Reply:
For me it's ok with what you do @zuk3975 :+1: